csharp/src/SeedLang/Runtime/VMState.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace SeedLang.Runtime {
+  // The state of SeedLang VM.
+  public enum VMState {
+    // The SeedLang VM is ready to execute a program.
+    Ready,
+    // The script is running.
+    Running,
+    // The execution is stopped temporarily.
+    Stopped,
```

Comment:
`Paused` is a better name than `Stopped` here.

Reply:
Fixed.